client/src/cl_main.cpp

```diff
@@ -1717,6 +1717,14 @@ bool CL_PrepareConnect()
 	}
 	else if (!ok && !missingfiles.empty() || cl_forcedownload)
 	{
+		if (::missingCommercialIWAD)
+		{
+			Printf(PRINT_WARNING,
+			       "Server requires commercial IWAD that was don't have.\n");
```

Comment:
"Server requires commercial IWAD that was don't have."

Should this be: "Server requires commercial IWAD that **we** don't have." ?

Reply:
Good eye.  Fixed.